### PR TITLE
Issue #142: Convert recent content block to a view.

### LINKED ISSUE #146: [D8] Convert "Recent content" block to a View
Followup to #40.
Relevant drupal.org issue: https://drupal.org/node/2020393

The new views block should contain:

- [x] All the same fields as the node module recent content block
- [x] Operations links moved into a drop-button column
- [x] a setting for how many items to display (for BC funcionality)
- [x] a new setting for content types to display based on https://github.com/backdrop/backdrop-issues/issues/4020

Advocate: @stpaultim

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/146

### CODE REVIEW

File: core/modules/node/config/views.view.content_recent.json [hunk 1]
```diff
@@ -0,0 +1,561 @@
+{
+    "_config_name": "views.view.content_recent",
+    "name": "content_recent",
+    "description": "Recent content.",
```
Comment (klonos) [inline core/modules/node/config/views.view.content_recent.json:4]: Let's retain the description of the legacy block (which should also be translatable):

```suggestion
    "_config_name": "views.view.content_recent",
    "_config_translatables": [
        "description"
    ],
    "name": "content_recent",
    "description": "A list of recently published content.",
```

File: core/modules/node/config/views.view.content_recent.json [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+                    "area_text_custom": {
+                        "id": "area_text_custom",
+                        "table": "views",
+                        "field": "area_text_custom",
+                        "relationship": "none",
+                        "group_type": "group",
+                        "ui_name": "",
+                        "label": "",
+                        "empty": true,
+                        "content": "No content available.",
```
Comment (klonos) [inline core/modules/node/config/views.view.content_recent.json:481]: ...this should also be in the translatable config.

**Comment by stpaultim** [comment]
Sandbox not working.

File: core/modules/node/config/views.view.content_recent.json [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+                        "relationship": "none",
+                        "group_type": "group",
+                        "ui_name": "",
+                        "operator": "in",
+                        "value": [],
+                        "group": "1",
+                        "exposed": true,
+                        "expose": {
+                            "operator_id": "type_op",
+                            "label": "Content types",
```
Comment (klonos) [inline core/modules/node/config/views.view.content_recent.json:411]:
```suggestion
                            "label": "Content types to display",
```

Should also be in the translatable configuration.

**Comment by stpaultim** [comment]
@jenlampton - Sandbox not working. Can you restart?

**Comment by klonos** [comment]
Closing in favor of https://github.com/backdrop/backdrop/pull/4596

**Pull request closed without merging**